demo/content/Form.js

```diff
@@ -84,7 +95,6 @@ export default class DemoContentForm extends Component {
                                  name="name.first"
                                  label="First Name"
                                  placeholder="Please enter your first name"
-                                 value="Jared"
                                  rules={['required']}
```

Comment:
Any reason you removed all examples of `defaultValue` and `value`? Could we add a couple examples using this?